cmdgad

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/python -B
-#!/usr/bin/env python -B
+#!/home/jzh/anaconda2/bin/python
```

Comment:
This path is specific to your own machine. Please update it to a generic path.

Also since you are using anaconda2, could you also add a section in README about installation and caveats (if any) of anaconda2?